app/main.py

```diff
@@ -1 +1,33 @@
-# write your code here
+class Animal:
+    alive: list["Animal"] = []
```

Comment:
```suggestion
    alive: list[Animal] = []
```
type annotations should represented by classes, not strings

Reply:
if you have problems with `type` is not defined, use
```
from __future__ imports annotations
```